### DISCUSSION TITLE
Lane guidance

### DISCUSSION THREAD

**Discussion by wavard** [devemux86/cruiser#110] [Q&A] [answered]
Will lane guidance be available in Cruiser?

**Comment by devemux86** [comment, marked as answer]
@wavard thanks for the suggestion.

It depends on whether such info is present in the calculated routes from the routers.

**Comment by wavard** [comment]
Great

Regards

Walter

**Comment by MotoUKRider** [comment]
This is a nice feature to have, I'm used to it from CoPilot.  It draws a simulation of the lanes I should take [with examples on this page called "Lane Assist"](https://support.copilotgps.com/en/support/solutions/articles/19000046170-how-do-i-setup-copilot-for-driving-) and makes messy junctions/merges quite straightforward.
